content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Product Update - v22.34
+description: Details of the product update v22.34 from 25th of August 2022.
+layout: rel
+version: "22.34"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-25
+redirect_from:
+  - /releases/22.34.html
+---
+
+## Improvements and Updates
+
+### Meaningful error for invalid SSH key entry
+
+When we introduced [support for newest SSH versions](/releases/22/14#ssh-keys-and-ssh-88)
+for the user SSH keys used to deploy components, we deprecated the old `ssh-dss`
+types. Now the user-interface shows a distinct error explaining the situation if
+the user enters an un-supported or malformed SSH Key instead of just disabling
+the Save button like it was before.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2234/RN2234-ssh-key-screen-error.png" title="SSH Key screen in case of Error" %}
+
+### Deployment of Externally Build Dockers
+
+Our team has just begun working on a new exciting feature to enable deployment of
+externally build docker images for the integration components. This feature might
```

Comment:
..buil**t**.. ..**of** integration components.
This capability will provide an alternative to Git push deployment of user developed components and is of interest to enterprise clients wishing to condense entire integration flows into single custom components and maintain control at the docker level.

Note - This is mambus stated goal of productising flows - versioning, scanning....etc. will all be done at the docker level. I hope this somewhat explains this approach..

Reply:
Thanks